docs/standard/io/file-path-formats.md

```diff
@@ -0,0 +1,171 @@
+author: "rpetrusha"
+ms.author: "ronpet"
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+# File path formats on Windows systems
+
+Members of many of the types in the <xref:System.IO> namespace include a `path` parameter that lets you specify an absolute or relative path to a file system resource. This path is then passed to [Windows file sysem APIs](https://msdn.microsoft.com/library/windows/desktop/aa364407(v=vs.85).aspx). This topic discusses the formats for file paths that you can use on Windows systems.
+
+## Traditional DOS paths
+
+A standard DOS path can consist of three components:
+
+- A volume or drive letter followed by the volume separator (`:`).
+- A directory name. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates subdirectories within the nested directory hierarchy.
+- An optional filename. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates the file path and the filename.
+
+If all three components are present, the path is absolute. If no volume or drive letter is specified and the directory names begins with the [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>), the path is relative from the root of the current drive is assumed. Otherwise, the path is relative to the current directory. The following table shows some possible directory and file paths.
+
+|Path  |Description  |
+| -- | -- |
+| C:\\Documents\Newsletters\Summer2018.pdf | An absolute file path from the root of drive C: |
+| \\Program Files\\Custom Utilities\\StringFinder.exe | An absolute path from the root of the current drive. |
+| 2018\\January.xlsx | A relative path to a file in a subdirectory of the current directory. |
+| ..\\Publications\\TravelBrochure.pdf | A relative path to file in a directory that is a peer of the current directory. |
+
+## UNC paths
+
+Universal naming convention (UNC) paths, which are used to access network resources, have the following format:
+
+- A server or host name, which is prefaced by \\\\. The server name can be a NetBIOS machine name or an IP/FQDN address (IPv4 as well as v6 are supported).
+- A share name, which is separated from the host name by \\. Together, the server and share name make up the volume.
+- A directory name. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates subdirectories within the nested directory hierarchy.
+- An optional filename. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates the file path and the filename.
+
+The following are some examples of UNC paths:
+
+|Path  |Description  |
+| -- | -- |
+| \\\\system07\\C$\\ | The root directory of the C: drive on `system07`. |
+| \\Server2\\Share\\Test\\Foo.txt | The Foo.txt file in the Test directory of the \\\\Server2\\Share volume.|
+
+UNC paths are always absolute. You can use relative paths only by mapping a UNC path to a drive letter.
```

Comment:
UNCs can use relative directory segments (`.`, `..`), but must be fully qualified.